src/industry_gui.cpp

```diff
@@ -1515,23 +1543,24 @@ enum CargoesFieldType {
 	CFT_HEADER,      ///< Header text.
 };
 
-static const uint MAX_CARGOES = 3; ///< Maximum number of cargoes carried in a #CFT_CARGO field in #CargoesField.
+static const uint MAX_CARGOES = 16; ///< Maximum number of cargoes carried in a #CFT_CARGO field in #CargoesField.
```

Comment:
True, it should really be max of town building accepts, industry accepts, industry produces, and 2 (because town buildings produce PASS and MAIL.) Let's just leave it.